html/semantics/scripting-1/the-script-element/moving-between-documents/resources/moving-between-documents-iframe.py

```diff
@@ -1,56 +1,59 @@
 import random
 import time
 
+from wptserve.utils import isomorphic_decode
+
+
 """
 This script serves
 """
 
 def main(request, response):
-  inlineOrExternal = request.GET.first("inlineOrExternal", "null")
-  hasBlockingStylesheet = request.GET.first("hasBlockingStylesheet", "true") == "true"
-  result = request.GET.first("result", "success")
-  type = "text/javascript" if request.GET.first("type", "classic") == "classic" else "module"
+  inlineOrExternal = request.GET.first(b"inlineOrExternal", b"null")
+  hasBlockingStylesheet = request.GET.first(b"hasBlockingStylesheet", b"true") == b"true"
+  result = request.GET.first(b"result", b"success")
+  type = b"text/javascript" if request.GET.first(b"type", b"classic") == b"classic" else b"module"
```

Comment:
If we use the `u` prefix for the "text/javascript" or "module" values, we don't need to use isomorphic_decode on `type` later.

Reply:
Yes. Updated the code. Thank you!